service_capacity_modeling/models/org/netflix/kafka.py

```diff
@@ -506,6 +496,27 @@ def default_desires(user_desires, extra_model_arguments: Dict[str, Any]):
         # write throughput * retention = usage
         state_gib = (write_bytes.mid * retention_secs) / GIB_IN_BYTES
 
+        # By supplying these buffers we can deconstruct observed utilization into
+        # load versus buffer.
+        buffers = Buffers(
+            default=Buffer(ratio=1.5),
+            desired={
+                # Amount of compute buffer that we need to reserve in addition to
+                # cpu_headroom_target that is reserved on a per instance basis
+                "compute": Buffer(ratio=1.5, components=[BufferComponent.compute]),
```

Comment:
ratio=2.5 here for tier0 and tier1 (to target 40% CPU utilization max on instance). It can be lower(1.5) for tier=2 and tier=3